src/test/a.act

```diff
@@ -0,0 +1,12 @@
+behaviour add of SafeAdd
```

Comment:
I think it's better if we make the filenames more clear. Maybe `add.act`?

Reply:
sorry, this file shouldn't be here. SafeAdd is already in `tests/safemath`. Will remove this